xcp_d/ingression/abcdbids.py

```diff
@@ -135,10 +135,14 @@ def convert_dcan_to_bids_single_subject(in_dir, out_dir, sub_ent):
                         ├── vent_2mm_<sub_id>_mask_eroded.nii.gz
                         └── wm_2mm_<sub_id>_mask_eroded.nii.gz
     """
-    assert isinstance(in_dir, str)
-    assert os.path.isdir(in_dir), f'Folder DNE: {in_dir}'
-    assert isinstance(out_dir, str)
-    assert isinstance(sub_ent, str)
+    if not isinstance(in_dir, str):
+        raise ValueError("in_dir must be a string")
+    if not os.path.isdir(in_dir):
+        raise ValueError(f'Folder does not exist: {in_dir}')
```

Comment:
```suggestion
        raise FileNotFoundError(f'Folder does not exist: {in_dir}')
```

I'd lean toward more specific exceptions.